exporter/datadogexporter/factory.go

```diff
@@ -39,7 +40,8 @@ const (
 	// typeStr is the type of the exporter
 	typeStr = "datadog"
 	// The stability level of the exporter.
-	stability = component.StabilityLevelBeta
+	stability     = component.StabilityLevelBeta
+	logsStability = component.StabilityLevelAlpha
```

Comment:
```suggestion
```
I would remove these. You are just renaming what are already constants, which makes things confusing. I know you are just extending what was already there, but IMHO it would be an improvement. It's not like it's used in multiple places.